<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackForm</class>
 <widget class="QDialog" name="StackForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(5,2,13);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>874</width>
        <height>680</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelGrafico">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/*background: rgb(250,250,250);*/
background: #9DAABC;</string>
     </property>
     <widget class="QPushButton" name="btnPush">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: rgb(254,215,215);
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>Push</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtInputDato">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
border:2px;
border-style:solid;
border-color:rgb(28,19,42);
border-radius: 8px;
}

QTextEdit:focus {
    background-color: #DDD;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPop">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: rgb(254,215,215);
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>Pop</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>401</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(5,2,13);</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 25pt &quot;Cooper Black&quot;;
color: white;</string>
       </property>
       <property name="text">
        <string>Stack</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="labelCodeImg">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>311</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
font: 14pt &quot;Cooper Black&quot;;
color: black;</string>
      </property>
      <property name="text">
       <string>Estructura del Nodo</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
